### ISSUE TITLE
Write docs

### ISSUE THREAD

**Issue by nicolo-ribaudo** [babel/babel-polyfills#4]
~I have to write these _before_ making this repo public~

Se docs are already in this repository, but I need to expand them for the first release.

**Comment by Fuzzyma** [comment]
Do you need help with this? I very much would love to have this released rather soon than later because I am delaying my latest release for it (trying to keep the bundle size down)

**Comment by nicolo-ribaudo** [comment]
Do you have any suggestion about how to improve https://github.com/babel/babel-polyfills/tree/master/docs?

I reopened this issue because I got feedback that there weren't enough docs, but I'm open to suggestions on what to add.

Maybe a migration guide would help?

**Comment by Fuzzyma** [comment]
Migration guide is imo one of the most important things you could have because literally everyone will come from the transform-runtime side and ideally dont want to spend more than 5min on migrating (sweet dreams :D).
I will have a closer look and report back. I actually tried using it already but couldnt get it to work

**Comment by nicolo-ribaudo** [comment]
@Fuzzyma WDYT? https://github.com/babel/babel-polyfills/blob/master/docs/migration.md

**Comment by nicolo-ribaudo** [comment]
Also https://github.com/babel/babel-polyfills/pull/15

**Comment by Fuzzyma** [comment]
Sorry for the late feedback. Was working ^^. I read through it and it is certainly a good start.
The design is just a bit messed up.

What I would like though is a short explanation what a config is actually doing. Even though people come from transform-runtime and core-js, they mostly never do what effect those configs actually have and try around until it just works (at least thats what I did a lot).

I also saw the readme was updated in #15. I like the introductianary explanation a lot.
I noticed that the link to the "usage" docs links to migration. So that would need to get fixed
